tests/printing/transformed/lazy-vals-legacy.flags

```diff
@@ -0,0 +1 @@
+-Ylegacy-lazy-vals
```

Comment:
Do we really need that? I thought `.flag` files in tests are deprecated and vulpix doesn't even take them into consideration

Reply:
It does work, the generated code differs.